SnaffCore/ActiveDirectory/AdData.cs

```diff
@@ -51,7 +51,33 @@ public List<string> GetDfsNamespacePaths()
         public DirectoryContext DirectoryContext { get; set; }
         private List<string> DomainControllers { get; set; } = new List<string>();
 
-        public AdData()
+        private string GetNetBiosDomainName(string dnsDomainName)
```

Comment:
I'm certain you could achieve this within the DirectorySearch() class that I had here previously - it's just another LDAP query, after all.

Reply:
I should be able to add a constructor to the DirectorySearch class it if is easier on the SOCKS plumbing.   I have to to go to the config partition but the existing constructor is hard-coded to go to the domain partition.